client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -46,9 +46,10 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <Typography className={classes.label}>E-MAIL ADDRESS</Typography>
```

Comment:
Use `sx={{ textTransform: 'uppercase' }}` 

A screen reader will read it out letter by letter as "E-M-A-I-L A-D-D-R-E-S-S" I would imagine a user with a screen reader would be confused with that. When we need all uppercase, **always use text transform**

Thanks! 👍🏻

Reply:
Change throughout